.github/workflows/ci.yml

```diff
@@ -40,14 +45,17 @@ jobs:
       - uses: coursier/setup-action@v1
         with:
           jvm: temurin:21
-      - run: sbt "test; scripted sbt-scalafix/* skip-windows/*"
```

Comment:
`sbt-1.5/*` was ignored on JDK21 until now. The fix was simply to bump a compatible scala 3 version (per the [compatibility matrix](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html)).